src/espcomp/mod.rs

```diff
@@ -0,0 +1,129 @@
+use std::path::PathBuf;
+
```

Comment:
In any case, I would strongly suggest to place the `espcomp` module under the `espidf` one, rather than creating a top-level one. and probably renaming it then to `components`. The import path for the user would then look like `embuild::espidf::components`, which looks sane.

Reply:
Yep, done.